Added localization support

tutte opened the pull request:
Added localization support for spanish and engish languages.

joemerlino commented:
@tutte did you fix this [issue](https://github.com/ZaidElkurdi/AssistantPlus/issues/1)? thanks in advance!

tutte commented:
I'm afraid i did not, I fixed the kSecAttrAccessibleWhenUnlocked error and tapping in a search result is perfectly working for me (iOS 8.4, iPhone 7,2). I also updated the project to 8.4 sdk. I would suggest you to compile and try!

ZaidElkurdi commented on spotifyPlugin/spotifyPlugin/Localization.m:
```diff
@@ -0,0 +1,24 @@
+    if (bundleString) {
+        return bundleString;
+    }
+    return localization_key;
```
Why return the key? I think it might be safer to always return bundleString instead of hiding a bug.

ZaidElkurdi commented on spotifyPlugin/spotifyPlugin/Resources/en.lproj/Localizable.strings:
```diff
@@ -0,0 +1,16 @@
+"REGEX_PLAY_BY" = "(?:.*)Play (.*) by (.*) on Spotify";
+"REGEX_PLAY_SONG" = "(?:.*)Play (.*)on Spotify";
+"SEARCHING" = "Searching...";
+"CATCH_SEARCH" = "busca";
```
Still in Spanish

ZaidElkurdi commented on spotifyPlugin/spotifyPlugin/SpotifyPlayCommands.m:
```diff
@@ -112,4 +112,13 @@ - (NSString*)getSongForArtist:(NSString*)artistName fromSongs:(NSArray*)songs {
 }
 
 
+-(NSString*)localizedStringForKey:(NSString*)string_key {
```
Why is this here as well as Localization.h?

ZaidElkurdi commented on spotifyPlugin/spotifyPlugin/SpotifySearchCommands.m:
```diff
@@ -13,13 +13,13 @@ @implementation SpotifySearchCommands
 - (BOOL)handleSpeech:(NSString *)text withTokens:(NSSet *)tokens withSession:(id<APSiriSession>)session {
   //Do a quick/rough check first to see if we should do any more thorough processing
-  if ([tokens containsObject:@"search"] && [tokens containsObject:@"on"] && [tokens containsObject:@"spotify"]) {
+  if ([Localization localized:@"CATCH_SEARCH"] && [Localization localized:@"CATCH_ON_SPOTIFY"]) {
```
Should be:
if ([tokens containsObject:[Localization localized:@"CATCH_SEARCH"]] && [tokens containsObject:[Localization localized:@"CATCH_ON_SPOTIFY"]]) {
